packages/edit-site/src/components/navigation-sidebar/navigation-panel/style.scss

```diff
@@ -96,18 +94,34 @@
 		text-align: left;
 		padding-left: $grid-unit-20;
 		padding-right: $grid-unit-20;
-		margin-bottom: $grid-unit-15;
 		color: inherit;
 	}
 }
 
 .edit-site-navigation-panel__template-item-title {
+	font-size: 14px;
```

Comment:
I'm no expert at handling css-in-js in the best way, but these styles are applied twice. Once here and once in css-in-js: `export const ItemTitleUI = styled( Text )` ([above changes)](https://github.com/WordPress/gutenberg/pull/29476/files#diff-8cd649731dc069d0516044e41cfadc7ff8e914d7fbc48d83996ab5878a798d3bR224)

Reply:
I am certainly no expert either, but I do not see these styles being applied twice in the Inspector. For some reason menu items sans description seem to be styled by the js, while the menu items with a description are styled by the css. So both were needed to make them consistent with one another.

Reply:
--cc @ItsJonQ for some css-in-js feedback :)

Reply:
@ntsekouras Haiii! If the results are working the way you're expecting (with the new font size, etc...), then you should be okay 👍 .